package.json

```diff
@@ -123,6 +123,14 @@
           "default": [],
           "markdownDescription": "List of packages you'd like to be left out of completions, symbol searches, and code actions.\n\nEx. `akka.actor.typed.javadsl` will ensure nothing in the `javadsl` package gets recommended to you.\n\nYou can find the list of default exclusions [here on the Metals website](https://scalameta.org/metals/docs/editors/new-editor.html#excluded-packages).\n\nIf you need to remove one of the defaults, you can simply include it and preface it with `--`."
         },
+        "metals.typeAnnotationsEnabled": {
+          "type": "boolean",
+          "markdownDescription": "When this option is enabled, for each method that can have infered types they are displayed either as additional decorations if they are supported for the editor or as info diagnostics otherwise."
```

Comment:
Okay so typo 'inferred'
But we can also be more specific, sth like:
`for each method that can have inferred types they are displayed as additional decorations.`